pkgs/development/ocaml-modules/mirage-time/unix.nix

```
@@ -1,11 +1,12 @@
-{ buildDunePackage, fetchurl, mirage-time, ocaml_lwt, duration }:
+{ buildDunePackage, fetchurl, mirage-time, lwt, duration }:
```

```suggestion
{ buildDunePackage, fetchurl, mirage-time, lwt, duration }:
```
lwt is at arguments but not added to `propagatedBuildInputs`.

https://github.com/ocaml/opam-repository/blob/master/packages/mirage-time-unix/mirage-time-unix.3.0.0/opam#L24

I get that this package will inherit `lwt` (indirectly) from `mirage-time`. Both ways will obviously work. But is there an ideal way to package this? Should we be listing all dependencies or rely on indirect dependencies?

---

Well spotted. This should be fixed now. I don’t know what is ideal.